CHANGES.md

```diff
@@ -1,9 +1,25 @@
 # Changes
 
 ### v0.4.6
-* Refactoring of metrics:
+* Refactor metrics:
   * Remove `GroupMetricResult` type in favour of a `Bunch`.
-  * Add `group_summary` transformers.
+  * Rename and slightly update signatures:
+    * `metric_by_group` changed to `group_summary`
+    * `make_group_metric` changed to `make_metric_group_summary`
+  * Add group summary transformers
+    `{difference,ratio,group_min,group_max}_from_group_summary`.
+  * Add factory `make_derived_metric`.
+* Add new metrics:
+  * base metrics `{true,false}_{positive,negative}_rate`
+  * group summary metrics `<metric>_group_summary`
+  * derived metrics `<metric>_{difference,ratio,group_min,group_max}`
+  * disparity metrics `{demographic_parity,equalized_odds}_{difference,ratio}`
+* Remove metrics:
+  * `fallout_rate` in favor of `false_positive_rate`
+  * `miss_rate` in favor of `false_negative_rate`
+  * `specificity_score` in favor of `true_negative_rate`
+* Change from public to private:
+  * `mean_{over,under}prediction` changed to `_mean_{over,under}prediction`
```

Comment:
Can I ask why, given that (I believe) these are used by the Dashboard?

Reply:
Re. dashboard--this specific plot was confusing to users, so we might rework it.

Re. why make it private--I looked around online and couldn't find a standard name for this metric. But I found something called `mean_overprediction_error` and `mean_underprediction_error` and their meaning is similar to ours, but very slightly different (e.g., overprediction is positive and underprediction is negative, and they differ in how they are normalized). So I would like to do a bit of asking around to figure what the right API should be. Since we're reworking so much about metrics, I think it makes sense to introduce this change now, so we have more freedom to make the API change in future.